container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -266,10 +272,8 @@ variables:
     default: 10950
 - name: DOMAIN
   options:
-    description: 'Base domain name of the UAA endpoint; `uaa.${DOMAIN}` must be correctly
-      configured to point to this UAA instance
-
-'
+    description: Base domain name of the UAA endpoint; `uaa.${DOMAIN}` must be correctly
+      configured to point to this UAA instance.
```

Comment:
Would this look better with folding (`>`) when it's multi-line?

Reply:
Yes. I just fixed it to look more like the rest of the descriptions and remove that `'` that made my brain stop for a couple seconds.